### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by kant**
on @266. Plus minor fixes (proposal)

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Some typo found (just only one). Plus minor fixes in the styling of the `readme.md` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ To build from source, you need the following installed and available in your $PA
 #### OS X Users
 Don't forget to install Java 7 or 8. You probably have 1.6.
 
-Export JAVA_HOME in order to use the supported Java version:
+Export `JAVA_HOME in order to use the supported Java version:
 ```sh
```
Comment (wing328) [inline README.md]: @kant Missing a closing "`"?

File: README.md [hunk 2]
```diff
@@ -789,7 +789,8 @@ Here are some companies/projects using Swagger Codegen in production. To add you
 - [Hewlett Packard Enterprise](https://hpe.com)
 - [High Technologies Center](http://htc-cs.com)
 - [Hootsuite](https://hootsuite.com/)
-- [Huawei Cloud](https://www.huaweicloud.com) - [Cloud Stream Service](http://www.huaweicloud.com/en-us/product/cs.html)
+- [Huawei Cloud](https://www.huaweicloud.com) 
+- [Cloud Stream Service](http://www.huaweicloud.com/en-us/product/cs.html)
```
Comment (wing328) [inline README.md]: I would keep it in 1 line or simply replace with the following:
```
- [Huawei Cloud](http://www.huaweicloud.com/en-us/product/cs.html)
```

File: README.md (hunk 1 above)
Comment (kant) [reply README.md]: Affirmatively, must be `JAVA_HOME`.

**Comment by wing328** [comment]
I've cherry-pick 62d76946a9e49bca198b1953e186bffbae7230c0. Closing this one.

Thanks for the enhancement.

**Pull request closed without merging**